lib/charms/opensearch/v0/opensearch_secrets.py

```diff
@@ -113,13 +113,7 @@ def _on_secret_changed(self, event: SecretChangedEvent):  # noqa: C901
 
         logger.debug("Secret change for %s", str(label_key))
 
-        # Leader has to maintain TLS and Dashboards relation credentials
```

Comment:
Updating the TLS certs after a secret_change event has been removed. New TLS certs should only be processed by the certificate_available event in order to make sure new CA's are recognized and the workflow for rotating the CA and renewing the certificates is being handled correctly.